Scripts/Python/grsnWallPython.py

```diff
@@ -667,9 +667,9 @@ def OnNotify(self,state,id,events):
                     except:
                         PtDebugPrint("grsnWallPython::OnNotify: Blocker not found on either panel")
                         return
-                if(team == kNorth and ageSDL["nState"][0] == kSetBlocker and cID == ageSDL["nChairOccupant"][0]):
+                if(team == kNorth and ageSDL["nState"][0] == kSetBlocker):
                     self.SetPanelBlocker(kNorth, index, not self.FindBlocker(kNorth, index))
-                if(team == kSouth and ageSDL["sState"][0] == kSetBlocker and cID == ageSDL["sChairOccupant"][0]):
+                if(team == kSouth and ageSDL["sState"][0] == kSetBlocker):
```

Comment:
Will this change allow players who are standing behind the occupant to operate the control panel?

Reply:
this was the wrong place to do this
Currently yes anyone that stands inside the region can use the panel buttons